shared/components/LogsLink/LogsLink.scss

```diff
@@ -0,0 +1,3 @@
+.logs-link {
+  align-self: center;
+}
```

Comment:
we should also add a question mark on hover, as everywhere else where we have tooltips